community/kernel/src/main/java/org/neo4j/kernel/internal/Version.java

```diff
@@ -36,6 +36,11 @@ public static String getKernelVersion()
         return getKernel().getVersion();
     }
 
+    public static String getNeo4jVersion()
+    {
+        return getKernel().getReleaseVersion();
```

Comment:
there are a lot of call to Version.getKernel().getReleaseVersion(), would propose we replace it with a new one to unify all of them and add some javadoc for this method what it will return (at least reference where it can be found)